Libraries/Components/ActivityIndicator/ActivityIndicatorIOS.ios.js

```diff
@@ -7,27 +7,18 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ActivityIndicatorIOS
- * @flow
```

Comment:
Had to disable flow for this file, passing props to ActivityIndicator with `{...this.props}` caused a bunch of these errors. Not sure if there's a better way to fix this but the component is deprecated and there's pretty much no code anymore there.
```
 62:       <ActivityIndicator
           ^ React element `ActivityIndicator`
 37:     animating: PropTypes.bool,
                    ^^^^^^^^^^^^^^ undefined. This type is incompatible with. See: Libraries/Components/ActivityIndicator/ActivityIndicator.js:37
 60:       animating: true,
                      ^^^^ boolean. See: Libraries/Components/ActivityIndicator/ActivityIndicator.js:60

```